microsetta_admin/templates/scan.html

```diff
@@ -2,5 +2,90 @@
 {% block content %}
 <h3>Microsetta Scan</h3>
 <div>
+    <form name="search_form" id="search_form" method="GET">
+        <table>
+            <tr>
+                <td><label for="sample_barcode">Barcode: </label></td>
+                <td><input type="text" name="sample_barcode" id="sample_barcode" {% if info %} value="{{info[0].barcode}}"{% endif %}></td>
+            </tr>
+            <tr>
+                <td></td>
+                <td><input type="submit" value="Find Sample"></td>
+            </tr>
+        </table>
+        {% if search_error %}
+            <p style="color:red">
+            {{search_error |e}}
+            </p>
+        {% endif %}
+        <script>
+           document.agForm.sample_barcode.focus()
+        </script>
+    </form>
+
+    <br>
+    <hr>
+    <br>
+
+    {% if info %}
+        <form action="/scan" name="update_form" id="update_form" method="post">
+        <input type="hidden" name="sample_barcode" value="{{info[0].barcode}}"/>
+        <table>
```

Comment:
It's not clear here how, in the case of a null sample site, a technician could differentiate whether the sample is unassigned, if there are no consents associated with the kit, or if the kit is unregistered. The type of follow up necessary differs

Reply:
Why is the technician making this determination and setting the sample_status rather than business logic within the application?  The application knows whether or not there is an account, whether or not there is a source etc etc.  Its possible to determine all of this based on the information being passed down over the admin api, but why involve the technician in deciding the status rather than just saying what the status is on this page?

Reply:
I like this. So present the technician with the systems assessment, and why, and allow them to in effect confirm?